app/assets/javascripts/filter_set.js.coffee

```diff
@@ -0,0 +1,27 @@
+class @FilterSet
+  constructor: (@$root, @key, @val) ->
+
+  $: (selector) ->
+    @$root.find(selector)
+
+  children: ->
+    @$("[data-filter-key=#{ @key }][data-filter-value=#{ @val }]")
+
+  adjacentChildren: ->
```

Comment:
hahaha just Googled "children of siblings of parent" to see if there was a word for it...yeeeeah :grin: 

`aria-owns` is interesting (didn't know about that before), but I think that's tangential.